java-client-model/src/main/java/com/cumulocity/rest/representation/cep/CepApiRepresentation.java

```diff
@@ -0,0 +1,16 @@
+package com.cumulocity.rest.representation.cep;
+
+import com.cumulocity.rest.representation.AbstractExtensibleRepresentation;
+
+public class CepApiRepresentation extends AbstractExtensibleRepresentation {
```

Comment:
is it needed in this module?

Reply:
Used in `PlatformApiRepresentation` - removing it from there would also mean API changes so I think it is safer to leave it here.